extensions/kube-object-event-status/tsconfig.json

```diff
@@ -13,6 +13,7 @@
     "esModuleInterop": true,
     "allowSyntheticDefaultImports": true,
     "experimentalDecorators": true,
+    "useDefineForClassFields": true,
```

Comment:
I thought you were getting rid of this change.

Reply:
Then I think we should do that in a separate PR before this one gets merged. It is a pretty big change in and of itself.

Reply:
I guess we should consider to keep this into current PR. The reasons:
1. It's part of migration guide for TS users https://mobx.js.org/migrating-from-4-or-5.html. And since we want to use decorators, we should include this.
2. It's impossible to satisfy TS compiler in current PR without `useDefineForClassFields`, so after merge `master` would be broken?

Reply:
@aleksfront what do you mean by after merge `master` would be broken? 
`useDefineForClassFields: true` at least needs to make work class field assignments without initial value, e.g.
class Test {
 @observable someVar: string; // no initial value, doesn't work with mobx-6
}